<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>854</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame_trackers">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>60</y>
      <width>254</width>
      <height>420</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QRadioButton" name="radioButton_KCF">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>251</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>KCF</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_CSRT">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>411</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>CSRT</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_MIL">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>261</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>MIL</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame_zoomedImage">
     <property name="geometry">
      <rect>
       <x>27</x>
       <y>160</y>
       <width>200</width>
       <height>200</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_select_algorithm">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Select an Algorithm</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_camera_feed">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>600</width>
      <height>420</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame_controls">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>854</width>
      <height>60</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QPushButton" name="button_selectROI">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Select ROI</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_fps">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_frametime">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>854</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
